### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_HighContrast-Black:In Get Signature Access window 'show date and time picker' element is not visible on applying High contrast theme.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5705]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG4.3.1; #ScreenReader; #Narrator; #FTP; #HighContrast;

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4.  Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. right click on blob and select "Get Shared Access Signature" control and hit enter.
6. Choose "High contrast-Black theme" in settings.
7. Check whether 'show date and time picker' option present in "Start time" and "Expiry time" field are MAS compliant.

## Actual results:
In High contrast-Black theme, 'show date and time picker' option present in the "Start time" and "Expiry time" fields are not visible.

**Same issue is observed with the following:**

- Manage Access Policies

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6. Choose "High contrast-Black theme" in settings.
7. Verify that all the elements of "Manage Access Policies" screen are accessible and MAS Compliant.

## Expected results:
Everything should be visible on applying High contrast-Black mode.

## User Impact:
Users might fail to acknowledge the presence of 'show date and time picker' option if it is not visible on applying high contrast black mode.

## Attachments:
![5705_A11y_Azure Storage Explorer_Desirable theme_With Normal Settings](https://user-images.githubusercontent.com/105198219/167816943-c4f09222-e6fc-4a48-b6e2-599140fa5c3c.png)

![5705_A11y_Azure Storage Explorer_On High contrast Black mode](https://user-images.githubusercontent.com/105198219/167817057-1f3e095b-d39e-4c55-95d6-3bcf853ee239.png)
![5705_AccessMangePolicie_NormalMode](https://user-images.githubusercontent.com/105198219/168088104-722fd968-f36d-4617-a959-b3e56e57a55a.png)
![5705_AccessManagePolicies_HighContrast-Black](https://user-images.githubusercontent.com/105198219/168088346-d46a08f6-23f1-49a4-bc21-5adbf78a1240.png)

**Comment by MRayermannMSFT** [comment]
Related #4173

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
Issue is not reproducing in the provided environment (ver 1.25.0). Hence closing the bug. Please see the below attachments.

In "Shared Access Signature" dialog box.
![5705_Issue not repro_Get shared signature](https://user-images.githubusercontent.com/105198219/175862114-4ac1c7f3-c65f-4824-a20d-4aee9137f9c8.png)

In "Manage Access Policies" dialog box.
![5705_Issue not repro_Manage Access Policies](https://user-images.githubusercontent.com/105198219/175862118-87c3f1a4-06c4-492e-8ba7-fa5b73f34169.png)

**Comment by swethakodigi** [comment]
#Closed

**Comment by swethakodigi** [comment]
GitHubTags:#Closed

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
